src/components/utils/rhf-inputs/error-inputs/error-input.js

```diff
@@ -32,11 +36,19 @@ const ErrorInput = ({ name, InputField }) => {
         return {};
     };
 
+    useEffect(() => {
+        if (isSubmitted && error && errorRef?.current) {
```

Comment:
Why `?` on `errorRef`, how can it be null ?

Reply:
because in the first render the errorRef will be null

Reply:
Current will be null, not errorRef